src/vcModals.cpp

```diff
@@ -563,6 +564,35 @@ void vcModals_DrawImageViewer(vcState *pProgramState)
   }
 }
 
+void vcModals_DrawProfile(vcState* pProgramState)
+{
+  const char* userName = pProgramState->settings.loginInfo.username;
+  const char* profile = vcString::Get("menuProfileTitle");
+
+  if (pProgramState->openModals & (1 << vcMT_Profile))
+    ImGui::OpenPopup(profile);
+
+  float width = 300.f;
+  float height = 250.f;
```

Comment:
I don't believe we normally have local variables like this, however, perhaps we should reconsider for the additional clarity in the UI code.

Reply:
Yeah, I said to @asunawu verbally that the UI tends to have magic numbers embedded in the ImGui calls but also that its not really standardised for UI magic numbers.

Reply:
Sounds good :+1: - I might just resolve this conversation since I didn't want this to be changed.